flink-sql-runner/src/main/java/com/github/streamshub/flink/InterpolatorImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.github.streamshub.flink;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClientBuilder;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class InterpolatorImpl implements Interpolator {
```

Comment:
I would have kept the original name really. `KubernetesSecretInterpolator` ?

Reply:
Yeah, that sounds better actually.